File: Sirma Academy May 2024/JavaScript/Conditional Statements/LeapYearChecker.js
function leapYearChecker(year) {
  if ((year % 4 === 0 && year % 100 !== 0) || year % 400 === 0) {
    return "It's a leap year";
  } else {
    return "It's not a leap year";
  }
}

console.log(leapYearChecker(1900));

File: Sirma Academy May 2024/Java/Conditional Statements/src/LeapYearChecker.java
import java.util.Scanner;

public class LeapYearChecker {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int inputYear = Integer.parseInt(sc.nextLine());

        if((inputYear % 4 == 0 && inputYear % 100 != 0) || inputYear % 400 == 0){
            System.out.println("It‘s a leap year!");
        } else {
            System.out.println("It‘s not a leap year.");

        }
    }
}
